sshtunnel.py

```diff
@@ -1111,10 +1123,13 @@ def _get_transport(self):
             self.logger.debug('Connecting via proxy: {0}'.format(proxy_repr))
             _socket = self.ssh_proxy
         else:
-            _socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            _socket = self._get_socket()
         if isinstance(_socket, socket.socket):
             _socket.settimeout(SSH_TIMEOUT)
-            _socket.connect((self.ssh_host, self.ssh_port))
+            address = [self.ssh_host, self.ssh_port]
+            if _socket.family == socket.AF_INET6:
+                address.extend([0, 0])
+            _socket.connect(tuple(address))
         transport = paramiko.Transport(_socket)
```

Comment:
At the moment it looks like the paramiko is supporting a tuple (address, port) argument. https://github.com/paramiko/paramiko/blob/dccc563/paramiko/transport.py#L395-L417
Probably it's easy to just pass `(self.ssh_host, self.ssh_port)` as an argument.